frontend/src/core/plural/components/PluralSelector.js

```diff
@@ -80,10 +80,10 @@ export class PluralSelectorBase extends React.Component<InternalProps> {
             <ul>
                 { locale.cldrPlurals.map((item, i) => {
                     return <li key={ item } className={ i === pluralForm ? 'active' : '' }>
-                        <a onClick={ () => this.selectPluralForm(i) }>
+                        <button onClick={ () => this.selectPluralForm(i) }>
                             <span>{ CLDR_PLURALS[item] }</span>
                             <sup>{ examples[item] }</sup>
-                        </a>
+                        </button>
```

Comment:
I'm curious what's the purpose of this change.

Reply:
React raised a warning, using a `<a>` with a `href` attribute is considered a bad pattern, and they are right: `<a>` should be links, leading to a different page or at least a URL change. In this case, it was not doing that, so it should have been a `<button>` from the start.